### PR TITLE
   notification plus suggestion

### PR DESCRIPTION
**Pull request by hananqwaider**
make suggestion and send it to admin paneel plus make a function to move icon inside header

### CODE REVIEW

File: public/js/index.js [hunk 1]
```diff
@@ -9,21 +9,25 @@ function initMap() {
...
-    setTimeout(function () {
-      isAnimate = false;
-      icon.classList.remove("icon-anim");
-    }, 10000);
-  }
-  buttons.forEach(function (button) {
-    button.addEventListener("click", iconAnim);
-  });
-})();
+// (function () {
```
Comment (abdhalees) [inline public/js/index.js]: remove commented code

File: public/js/index.js [hunk 2]
```diff
@@ -9,21 +9,25 @@ function initMap() {
...
+//       isAnimate = false;
+//       icon.classList.remove("icon-anim");
+//     }, 7000);
+//   }
+//   buttons.forEach(function (button) {
+//     button.addEventListener("click", iconAnim);
+//   });
+// })();
+const order = document.getElementById('submit');
+order.addEventListener('click',(e)=>{
```
Comment (abdhalees) [inline public/js/index.js]: what does this function do?

File: src/controllers/homepage.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ const POSThomepage =(req,res)=>{
         console.log(err);
```
Comment (abdhalees) [inline src/controllers/homepage.js]: remove log and handle errors properly

File: src/controllers/suggestion.js [hunk 4]
```diff
@@ -1,4 +1,18 @@
+const queries = require('../model/query/queries');
 const suggestion = (req,res)=>{
   res.status(200).render('suggestion');
 }
-module.exports={suggestion}
+const makesuggestion = (req,res) =>{
+const {suggestion} = req.body;
+queries.makesuggestion(req.user.id,suggestion,(errorConnectingToDB , Result) => {
+  if (errorConnectingToDB) {
+    res.redirect('/err');
```
Comment (abdhalees) [inline src/controllers/suggestion.js]: do you have this route?

File: public/js/index.js (hunk 2 above)
Comment (hananqwaider) [reply public/js/index.js]: we put it for vaibertion  and test it but it doesn't work so i will delete it now

File: public/js/index.js (hunk 1 above)
Comment (hananqwaider) [reply public/js/index.js]: hader

**Pull request closed without merging**